src/ladds/Simulation.cpp

```diff
@@ -133,6 +133,39 @@ Simulation::initIntegrator(AutoPas_t &autopas, ConfigReader &config) {
   return std::make_tuple<>(std::move(csvWriter), std::move(accumulator), std::move(integrator));
 }
 
+std::tuple<size_t, std::shared_ptr<HDF5Writer>, std::shared_ptr<ConjuctionWriterInterface>> Simulation::initWriter(
+    ConfigReader &config) {
+  if (config.defines("io/hdf5")) {
+    std::shared_ptr<HDF5Writer> hdf5Writer;
+    const auto hdf5WriteFrequency = config.get<unsigned int>("io/hdf5/writeFrequency", 0u);
+    const auto hdf5CompressionLvl = config.get<unsigned int>("io/hdf5/compressionLevel", 0u);
+    // either append to checkpoint ...
+    const auto checkpointFileName = config.get<std::string>("io/hdf5/checkpoint/file", "");
```

Comment:
So, in the default config this is currently under io/checkpoint/file ( https://github.com/esa/LADDS/blob/42ace8a9851e762742c61197eb8745f4b2e05a92/cfg/default_cfg.yaml#L36 )

Reply:
fixed in e2873be